### PR TITLE
[dom-gpu] nvprof

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix https://jira.cscs.ch/browse/MAINT-87

### CODE REVIEW

File: jenkins-builds/7.0.UP00-19.03-mc [hunk 1]
```diff
@@ -59,6 +59,7 @@
  mpiP-57fc864-CrayGNU-19.03.eb                      --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
  mpiP-57fc864-CrayIntel-19.03.eb                                         --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
  MUST-v1.6-rc1-CrayGNU-19.03.eb                     --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ nvidia-nsight-2019.3.1.eb                          --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
```
Comment (kraushm) [inline jenkins-builds/7.0.UP00-19.03-mc]: Does nsight also work on CPU only applications?

Comment (jgphpc) [reply jenkins-builds/7.0.UP00-19.03-mc]: maybe but with https://developer.nvidia.com/nsight-systems

Comment (jgphpc) [reply jenkins-builds/7.0.UP00-19.03-mc]: nvprof is on the $PATH when `cudatoolkit` is loaded.

Comment (jgphpc) [reply jenkins-builds/7.0.UP00-19.03-mc]: ah nvtop

Comment (kraushm) [reply jenkins-builds/7.0.UP00-19.03-mc]: Check for nvtop. Regarding `nsight`, this installation is not for `nsight-systems`, is it? Does it make sense then to add it to the `mc` production list?

Comment (jgphpc) [reply jenkins-builds/7.0.UP00-19.03-mc]: let's focus on gpu first + i will do another PR for nsight-systems if needed

**Pull request merged**